docs/development/l10n.rst

```diff
@@ -91,8 +91,193 @@ by clicking on it:
 
 |Weblate translate context screenshot|
 
+Glossary
+--------
+
 A :doc:`glossary <../terminology>` is available to explain the
-SecureDrop specific terms.
+SecureDrop specific terms. In addition, it is important that some key
+term are understood and precisely translated. You will find some of
+them as well as their translations in the `EFF glossary`_.
```

Comment:
Ah, so it looks like these terms are from the EFF glossary (which is fine, just want the credit to be crystal clear). Let's directly say "The general security terms below are from the EFF glossary (link). At the EFF glossary (link) you can also find the translations for these terms in many languages.".

Reply:
This may get complicated as the glossary grows with terms from other sources. What about linking the term definition to the original EFF definition ? I'll update with this so you can see what I mean.